device-types/SNR/S2989G-48TX-POE.yml

```diff
@@ -0,0 +1,331 @@
+---
+manufacturer: SNR
+model: S2989G-48TX-POE
+slug: snr-s2989g-48tx-poe
+part_number: S2989G-48TX-POE
+u_height: 1.0
+is_full_depth: false
+airflow: right-to-left
+comments: '[SNR-S2989G-24TX](https://snr.systems/ru/catalog/setevoe-oborudovanie/kommutatory-dostupa/snr-s2989g/snr-s2989g-24tx)'
+console-ports:
+  - name: con0
+    label: CONSOLE
+    type: rj-45
+power-ports:
+  - name: power AC
+    type: iec-60320-c14
+    maximum_draw: 790
+power-outlets:
+  - name: usb
```

Comment:
USB-a ports as power-outlets is not something currently being modeled in this repo